Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js

```diff
@@ -0,0 +1,33 @@
+var totalTime;
+var curTime;
+
+var setButton = document.getElementById("setload");
+var loadingBar = document.getElementById("progress");
```

Comment:
I am pretty sure this could be a const instead of var.

Reply:
Actually both of these could be.

Reply:
Oh good catch!

Reply:
Yes, and I would stay away from `var`. The `var` declaration is scoped to it's nearest function (in this case it's the global context `window`). `let` and `const` are scoped to their code blocks, e.g., if statement, for loop, `{}`.